From 92b7f81e1dde7c760327d0cc2170d6ce225e093b Mon Sep 17 00:00:00 2001
From: Dusty Mabe <dusty@dustymabe.com>
Date: Tue, 30 Jan 2024 13:39:46 -0500
Subject: [PATCH 2/2] stages(sgdisk): support label option

We only support `gpt` here so it would seem this option doesn't
make much sense to add, but it will make it so that the mpp-define-images
from osbuild-mpp can be passed in to `org.osbuild.sgdisk` too as well
as `org.osbuild.sfdisk`.
---
 stages/org.osbuild.sgdisk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/stages/org.osbuild.sgdisk b/stages/org.osbuild.sgdisk
index 1627a90f..f85a762a 100755
--- a/stages/org.osbuild.sgdisk
+++ b/stages/org.osbuild.sgdisk
@@ -35,6 +35,11 @@ SCHEMA_2 = r"""
       "type": "boolean",
       "default": true
     },
+    "label": {
+      "description": "The type of the partition table. Only 'gpt' supported.",
+      "type": "string",
+      "enum": ["gpt"]
+    },
     "partitions": {
       "description": "Partition layout ",
       "type": "array",
-- 
2.43.0

